index/index.rst

```diff
@@ -30,7 +30,7 @@ This documents the *latest* ownCloud release, suitable for production use.
 ownCloud Server 6.0
 -------------------
 
-This documents the *previous* ownCloud release, suitable for production use.
+This documents the *older* ownCloud release, suitable for production use.
```

Comment:
Guys, you realize that this pull-request is outdated? :)

Btw: I undestood that 6.0 was deprecated with the release of 8.0. I'm infinitely puzzled.